bucket/openOCD.json

```diff
@@ -1,10 +1,15 @@
 {
-    "version": "20201228",
+    "version": "0.11.0",
     "description": "OpenOCD is an open-source tool that allows debugging various ARM devices with GDB using a wide variety of JTAG programmers.",
-    "homepage": "https://gnutoolchains.com/arm-eabi/openocd/",
-    "license": "Unknown",
-    "url": "https://sysprogs.com/getfile/1364/openocd-20201228.7z",
-    "hash": "2a45b7fee2974c36650c5bae95daa9b577c2241557332818f08e27a364de4311",
-    "extract_dir": "OpenOCD-20201228-0.10.0",
-    "env_add_path": "bin"
+    "homepage": "https://openocd.org/",
+    "license": "GPL-2.0-or-later",
+    "url": "https://github.com/openocd-org/openocd/releases/download/v0.11.0/openocd-v0.11.0-i686-w64-mingw32.tar.gz",
+    "hash": "b195e3c3e9fe3f4fe9dfd9780ff08336ef8ed3015a76cc883d9e21e57c456f33",
+    "bin": "bin\\openocd.exe",
```

Comment:
Can you verify once if the file is 32 or 64 bit?

The URL mentioning w64 is a bit wierd.

I am not sure if it means mingw-w64 or Windows 64.

Reply:
Seems like the distributed version is 32 bit executable (openocd.exe). I don't know why it's named as w64 :)